### DISCUSSION TITLE
How to convert the fp32 to fp16 models?

### DISCUSSION THREAD

**Discussion by Amin456789** [openvinotoolkit/openvino#20745] [Q&A] [answered]
Greetings!

I would like to convert a f32 to a f16 for a lower size for my better ram usage 
there is a model that people from here are using for their cpu version of LCM stable diffusion:
https://github.com/rupeshs/fastsdcpu

the model can be found here;
https://huggingface.co/deinferno/LCM_Dreamshaper_v7-openvino

could someone please convert this model to a f16 model and share it with us? or can someone please tell me how to do it?

kind regards

**Comment by ilya-lavrenov** [comment, marked as answer]
Hi @Amin456789 
Please, use OpenVINO conversion tool to convert model to FP16 https://docs.openvino.ai/2023.1/openvino_docs_model_processing_introduction.html
This tool saves model to FP16 by default (`compress_to_fp16` is `True` by default)

**Comment by Amin456789** [comment]
thank u
